src/utils/getSelectedEpisodeOrSearchedEpisodes.ts

```diff
@@ -0,0 +1,19 @@
+import { IEpisode } from "../types";
+
+export function getSelectedEpisodeOrSearchedEpisodes(
+  episodeID: number | null,
+  filteredEpisodes: IEpisode[]
+): IEpisode[] {
+  if (episodeID === null) {
+    return filteredEpisodes;
+  } else {
+    const selectedEpisode: undefined | IEpisode = filteredEpisodes.find(
```

Comment:
Could you use filter here instead of find to achieve the same output?
Using filter would mean that you don't have to initialise an empty array, nor push to it